src/sadari/InputView.java

```diff
@@ -0,0 +1,24 @@
+package sadari;
+
+import java.util.Scanner;
+
+public class InputView {
+    private Scanner scn = new Scanner(System.in);
```

Comment:
```suggestion
    private Scanner scanner = new Scanner(System.in);
```

특별한 이유 없다면 줄여쓰기는 지양해 주세요.